ci/cloudbuild/builds/coverage.sh

```diff
@@ -76,27 +76,27 @@ time {
 }
 
 codecov_args=(
-  "-X" "gcov"
-  "-f" "${MERGED_COVERAGE}"
-  "-q" "${HOME}/coverage-report.txt"
-  "-B" "${BRANCH_NAME}"
-  "-C" "${COMMIT_SHA}"
-  "-P" "${PR_NUMBER:-}"
-  "-b" "${BUILD_ID:-}"
+  "--filter=gcov"
```

Comment:
Looks like `-X` maps to `--feature`.... but maybe we want to replace the whole line with just `--gcov` ?

https://docs.codecov.com/docs/codecov-uploader#uploader-command-line-arguments

Reply:
I actually tried that, but it appears to mean something different.